{"messages":[{"error_id":"Release","references":[{"column":9,"file":"/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/conflict/007-MPI-conflict-get-get-local-yes.c","line":41},{"column":9,"file":"/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/conflict/007-MPI-conflict-get-get-local-yes.c","line":39}],"text":"Found call to MPI_Get at /home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/conflict/007-MPI-conflict-get-get-local-yes.c:41:9 which is in conflict with MPI_Get at /home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/conflict/007-MPI-conflict-get-get-local-yes.c:39:9 before release","type":"Local Data Race - Local write by call"}]}
